examples/style_sheets/ggplot.py

```diff
@@ -8,7 +8,7 @@
 
 These settings were shamelessly stolen from [1]_ (with permission).
 
-.. [1] http://www.huyng.com/posts/sane-color-scheme-for-matplotlib/
```

Comment:
This site appears to be down, so I figured the next best thing was the newest copy on `archive.org`